DigitalLearningSolutions.Data/Services/SessionService.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System.Data;
+    using Dapper;
+
+    public interface ISessionService
+    {
+        int StartOrRestartSession(int candidateId, int customisationId);
+        void StopSession(int candidateId);
+        void UpdateSessionDuration(int sessionId);
+    }
+
+    public class SessionService : ISessionService
+    {
+        private readonly IDbConnection connection;
+
+        public SessionService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        private const string stopSessionsSql =
+            @"UPDATE [Sessions] SET [Active] = 0
+              WHERE [CandidateId] = @candidateId;";

```

Comment:
Minor but I don't think the square brackets are necessary? Shouldn't cause an issue but might be worth removing for consistency 🙂

Reply:
Yep, removed nearly all of them in a4825ca 🙂 The only one I left was `[SessionID]`, which doesn't strictly need the square brackets but it is very similar to the keyword `Session_ID` so I think it is worth keeping the square brackets to stop any accidents from happening. I can remove that pair as well if you would prefer 🙂